source/doc-odm-user-guide/quickstart_user.rst

```diff
@@ -18,19 +18,19 @@ Study Browser
 -------------
 
 .. image:: images/quickstart_user_studybrowser.png
```

Comment:
probably we should change the screenshot to the new one where data class facets are presented instead of signal type facets|?